### PR TITLE
EDU-1144 Redesign navigation bar

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1144

### CODE REVIEW

File: src/components/navigation-utils.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import routes from '../utils/routes.js';
+import EditIcon from './icons/general/edit-icon.js';
+import UsersIcon from './icons/main-menu/users-icon.js';
+import SettingsIcon from './icons/main-menu/settings-icon.js';
+import DashboardIcon from './icons/main-menu/dashboard-icon.js';
+import { ControlOutlined, QuestionOutlined } from '@ant-design/icons';
+import permissions, { hasUserPermission } from '../domain/permissions.js';
+
+export const getCommonNavigationMenuItems = ({ t, user, helpPage }) => {
```
Comment (ahelmberger) [inline src/components/navigation-utils.js:10]: Thank God we finally do not have to repeat this in all apps anymore :heart:

Comment (adrianaluchian) [reply src/components/navigation-utils.js:10]: Yeees! 🕯️ ✝️

**Comment by ahelmberger** [comment]
Looks really great and works as expected :heart_eyes:

**Review by ahelmberger — APPROVED** [review]

**Comment by adrianaluchian** [comment]
> Looks really great and works as expected 😍

Perfect! 🙇

**Pull request merged**